### PR TITLE
Add vallox set_profile service documentation

### PR DESCRIPTION
**Pull request by treetip**
## Proposed change

Add vallox set_profile service documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#120225
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Style**
  - Enhanced visual elements with margin adjustments, new backgrounds, and flex displays for improved UI consistency.
  - Added icons to various section titles for better visual cues and accessibility.
  - Updated button styles with new alignments, colors, and border-radius for a modernized look.
  - Enhanced footer layout with max-width and padding updates.

- **Documentation**
  - Updated and clarified documentation, including correcting the spelling of "Aprilaire" to "AprilAire" and refining setup instructions.
  - Added notes to integrations, emphasizing requirements and supported features.
  - Introduced documentation for the new action `vallox.set_profile`, detailing parameters for profile management.

- **Bug Fixes**
  - Fixed link updates, ensuring correct URLs and proper navigation within documentation.

- **New Features**
  - Introduced new integration switches and services, such as "Auto-charge mode" for vehicles and `vallox.set_profile` in the Vallox integration, enhancing profile management capabilities for users.
  
- **Refactor**
  - Simplified and updated onboarding and configuration guides to streamline user experience and provide clearer setup instructions.

### CODE REVIEW

**Review by slovdahl — APPROVED** [review]

File: source/_integrations/vallox.markdown [hunk 1]
```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
 | Service data attribute | Optional | Description                                     |
 |------------------------|----------|-------------------------------------------------|
 | `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### Service `vallox.set_profile`
```
Comment (c0ffeeca7) [inline source/_integrations/vallox.markdown]:
```suggestion
### Action `vallox.set_profile` 
```
In this release, we are renaming 'service' to 'action'

File: source/_integrations/vallox.markdown [hunk 2]
```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
 | Service data attribute | Optional | Description                                     |
 |------------------------|----------|-------------------------------------------------|
 | `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### Service `vallox.set_profile` 
+
+Set the profile, and optionally a duration for the profile to be active.
+
+| Service data attribute | Optional | Description                                                                                                                                                                          |
```
Comment (c0ffeeca7) [inline source/_integrations/vallox.markdown]:
```suggestion
| Data attribute | Optional | Description                                                                                                                                                                          |
```

File: source/_integrations/vallox.markdown [hunk 3]
```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
...
+|------------------------|---------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `profile`              |       no | Profile to set, one of `Home`, `Away`, `Boost`, `Fireplace` or `Extra`.                                                                                                              |
```
Comment (c0ffeeca7) [inline source/_integrations/vallox.markdown]:
```suggestion
| `profile`              |       no | Profile to set, one of `Home`, `Away`, `Boost`, `Fireplace`, or `Extra`.                                                                                                              |
```

File: source/_integrations/vallox.markdown [hunk 4]
```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
...
+| `duration`             |      yes | Duration to activate the profile for in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. |
```
Comment (c0ffeeca7) [inline source/_integrations/vallox.markdown]:
```suggestion
| `duration`             |      yes | Duration to activate the profile for; in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. | 
```

File: source/_integrations/vallox.markdown [hunk 5]
```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
...
 | `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### Action `vallox.set_profile` 
+
+Set the profile, and optionally a duration for the profile to be active.
+
+| Data attribute | Optional | Description                                                                                                                                                                          |
+|------------------------|---------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `profile`              |       no | Profile to set, one of `Home`, `Away`, `Boost`, `Fireplace`, or `Extra`.                                                                                                              |
+| `duration`             |      yes | Duration to activate the profile for; in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. |
```
Comment (c0ffeeca7) [inline source/_integrations/vallox.markdown]:
```suggestion
| `duration`             |      yes | Duration to activate the profile for; in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. | 
```
should we say that 65535 is about 45.5 days?

Comment (treetip) [reply source/_integrations/vallox.markdown]: 65535 is actually a special case to set the profile permanently

Comment (c0ffeeca7) [reply source/_integrations/vallox.markdown]: I see. Could you add a note to the description?

Comment (treetip) [reply source/_integrations/vallox.markdown]: it's the last part

the code changed a bit, i'll try to update this later today to match that and get this merged

Comment (c0ffeeca7) [reply source/_integrations/vallox.markdown]: Oh dear. My oversight. Thank you.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @treetip :+1:

**Pull request merged**